src/javascript/katex-entry.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Entry point to auto-render KaTeX in the article once loaded.
+ */
+var id = setInterval(function() {
```

Comment:
To avoid polluting global scope, maybe wrap this whole thing in a `(function() { ... })()`?